next/src/components/sections/NewsletterSection/NewsletterSection.tsx

```diff
@@ -0,0 +1,175 @@
+    }
+    if (!VALID_EMAIL_FORMAT.test(String(email).toLowerCase())) {
+      return { isValid: false, error: t('NewsletterSection.error.emailIncorrectFormat') }
+    }
+
+    return { isValid: true }
+  }
+
+  const newsletterEndpointsMap: Record<Enum_Componentsectionsnewsletter_Newslettertype, string> = {
+    starz: '/api/newsletter-starz/subscribe',
+  }
+
+  const subscribeToNewsletterMutation = useMutation({
+    mutationFn: (emailToSubscribeeeee: string) =>
+      axios.post(
+        newsletterEndpointsMap[newsletterType],
+        {
+          email: emailToSubscribeeeee,
+        },
+        {
+          headers: {
+            'Content-Type': 'application/json',
+          },
+        },
+      ),
+    onSuccess: () => {
+      clearMessages()
+      setSuccessMessage(t('NewsletterSection.success'))
+    },
+    onError: (error) => {
+      // eslint-disable-next-line no-console
+      console.error(error)
+      clearMessages()
+      setErrorMessage(t('NewsletterSection.error.somethingWentWrong'))
+    },
+  })
+
+  const handleSubmit = async (submittedEmail: string) => {
+    clearMessages()
+
+    if (!validateForm().isValid) {
+      setErrorMessage(validateForm().error)
+      inputRef?.current?.focus()
+
+      return
+    }
+
+    setStatusMessage(t('NewsletterSection.sending'))
+
+    subscribeToNewsletterMutation.mutate(submittedEmail)
+  }
+
+  return (
+    <SectionContainer className="lg:py-24">
+      <div className="flex flex-col items-start justify-between gap-8 lg:flex-row lg:gap-34">
+        <div className="flex flex-col gap-6 lg:gap-10">
+          <SectionHeader title={title} text={text} />
+          <div className="flex flex-col gap-4">
+            <Typography variant="h5" as="p" className="font-semibold">
+              {socialLinksTitle}
+            </Typography>
+            <div className="flex flex-wrap gap-3">
+              {facebookUrl ? (
+                <Button
+                  href={facebookUrl}
+                  icon={<FacebookIcon />}
+                  aria-label="Facebook"
+                  target="_blank"
+                  variant="icon-wrapped"
+                  rel="noreferrer"
+                  hasLinkIcon={false}
+                  className="size-10 bg-background-passive-primary lg:size-12"
+                />
+              ) : null}
+              {instagramUrl ? (
+                <Button
+                  href={instagramUrl}
+                  icon={<InstagramIcon />}
+                  aria-label="Instagram"
+                  target="_blank"
+                  variant="icon-wrapped"
+                  rel="noreferrer"
+                  hasLinkIcon={false}
+                  className="size-10 bg-background-passive-primary lg:size-12"
+                />
+              ) : null}
+            </div>
+          </div>
+        </div>
+        <div className="flex grow flex-col gap-4 lg:min-w-100 lg:gap-6 lg:rounded-lg lg:border lg:border-border-passive-primary lg:p-8">
```

Comment:
`lg:border-border-passive-primary` -> It should not be needed to specify border color, it' should get default color automatically.